recipes-qt/qt5/qtbase_git.bb

```diff
@@ -119,6 +119,7 @@ PACKAGECONFIG[xkb] = "-xkb,-no-xkb -no-xkbcommon,libxkbcommon"
 PACKAGECONFIG[xkbcommon] = "-xkbcommon,-no-xkbcommon,libxkbcommon,xkeyboard-config"
 PACKAGECONFIG[evdev] = "-evdev,-no-evdev"
 PACKAGECONFIG[mtdev] = "-mtdev,-no-mtdev,mtdev"
+PACKAGECONFIG[lttng] = "-trace lttng,,lttng-ust"
```

Comment:
I'll add "-trace no" to explicitly disable it when not enabled.

That's what we're using:
https://github.com/webosose/meta-webosose/blob/master/meta-webos/recipes-qt/qt5/qtbase_git.bbappend#L53